<ui version="4.0" >
 <author>Jeffrey Lipton</author>
 <class>MotorHead</class>
 <widget class="QMainWindow" name="MotorHead" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset>Cornell seal.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTabWidget" name="Tabby" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>661</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="MotorTab" >
     <attribute name="title" >
      <string>Motor Testing</string>
     </attribute>
     <widget class="QLabel" name="label_responses" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>40</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Program Responses</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy" >
       <cstring>Responses</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="CommandLine" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>440</y>
        <width>631</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_motors" >
      <property name="geometry" >
       <rect>
        <x>490</x>
        <y>40</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="text" >
       <string>Motors avalible</string>
      </property>
      <property name="textFormat" >
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy" >
       <cstring>MotorsTree</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="saveras" >
      <property name="geometry" >
       <rect>
        <x>580</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save Log As</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saver" >
      <property name="geometry" >
       <rect>
        <x>490</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ChangeFile" >
      <property name="geometry" >
       <rect>
        <x>400</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Change Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReStart" >
      <property name="geometry" >
       <rect>
        <x>110</x>
        <y>480</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Restart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearLog" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Clear Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mover" >
      <property name="geometry" >
       <rect>
        <x>620</x>
        <y>320</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Move</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Responses" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>60</y>
        <width>431</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="EStop" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>520</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Emergency STOP</string>
      </property>
     </widget>
     <widget class="QSlider" name="MoveBar" >
      <property name="geometry" >
       <rect>
        <x>520</x>
        <y>320</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="Positioner" >
      <property name="geometry" >
       <rect>
        <x>470</x>
        <y>320</y>
        <width>46</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="MotorsTree" >
      <property name="geometry" >
       <rect>
        <x>480</x>
        <y>60</y>
        <width>201</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Graphing" >
     <attribute name="title" >
      <string>Graphing</string>
     </attribute>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>650</x>
        <y>10</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Max Value</string>
      </property>
      <property name="buddy" >
       <cstring>MaxVal</cstring>
      </property>
     </widget>
     <widget class="QSlider" name="MaxVal" >
      <property name="geometry" >
       <rect>
        <x>670</x>
        <y>30</y>
        <width>21</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>580</x>
        <y>10</y>
        <width>59</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Min Value</string>
      </property>
      <property name="buddy" >
       <cstring>MinVal</cstring>
      </property>
     </widget>
     <widget class="QSlider" name="MinVal" >
      <property name="geometry" >
       <rect>
        <x>610</x>
        <y>30</y>
        <width>21</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame" >
      <property name="windowModality" >
       <enum>Qt::NonModal</enum>
      </property>
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>10</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <property name="cursor" >
       <cursor>2</cursor>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth" >
       <number>8</number>
      </property>
      <widget class="QGraphicsView" name="PictureTube" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>391</width>
         <height>261</height>
        </rect>
       </property>
       <property name="cursor" >
        <cursor>2</cursor>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>580</x>
        <y>170</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Zoom</string>
      </property>
      <property name="buddy" >
       <cstring>Zoomer</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveImage" >
      <property name="geometry" >
       <rect>
        <x>590</x>
        <y>250</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>500</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Zoom</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Element" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber" >
      <property name="geometry" >
       <rect>
        <x>680</x>
        <y>190</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="segmentStyle" >
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="MaxValeDisplay" >
      <property name="geometry" >
       <rect>
        <x>660</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="segmentStyle" >
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="MinValeDisplay" >
      <property name="geometry" >
       <rect>
        <x>590</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="segmentStyle" >
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Element Selection </string>
      </property>
      <property name="buddy" >
       <cstring>Element</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="label_6" >
      <property name="geometry" >
       <rect>
        <x>500</x>
        <y>300</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7" >
      <property name="geometry" >
       <rect>
        <x>560</x>
        <y>300</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9" >
      <property name="geometry" >
       <rect>
        <x>600</x>
        <y>310</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Step Size</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Namer" >
      <property name="geometry" >
       <rect>
        <x>600</x>
        <y>330</y>
        <width>23</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5" >
      <property name="geometry" >
       <rect>
        <x>440</x>
        <y>300</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TableMove" >
      <property name="geometry" >
       <rect>
        <x>600</x>
        <y>370</y>
        <width>75</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text" >
       <string>Move</string>
      </property>
     </widget>
     <widget class="QSlider" name="Zoomer" >
      <property name="geometry" >
       <rect>
        <x>580</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox" >
      <property name="geometry" >
       <rect>
        <x>240</x>
        <y>500</y>
        <width>46</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="VidZoom" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>500</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="Video" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>300</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="cursor" >
       <cursor>2</cursor>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Stepper" >
      <property name="geometry" >
       <rect>
        <x>630</x>
        <y>330</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinX" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>490</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep" >
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinY" >
      <property name="geometry" >
       <rect>
        <x>490</x>
        <y>490</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep" >
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinZ" >
      <property name="geometry" >
       <rect>
        <x>550</x>
        <y>490</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep" >
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QSlider" name="X" >
      <property name="geometry" >
       <rect>
        <x>440</x>
        <y>320</y>
        <width>25</width>
        <height>161</height>
       </rect>
      </property>
      <property name="singleStep" >
       <number>1</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition" >
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval" >
       <number>10</number>
      </property>
     </widget>
     <widget class="QSlider" name="Y" >
      <property name="geometry" >
       <rect>
        <x>500</x>
        <y>320</y>
        <width>25</width>
        <height>161</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition" >
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval" >
       <number>10</number>
      </property>
     </widget>
     <widget class="QSlider" name="Z" >
      <property name="geometry" >
       <rect>
        <x>560</x>
        <y>320</y>
        <width>25</width>
        <height>161</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition" >
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval" >
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Konsole" >
     <attribute name="title" >
      <string>Konsole</string>
     </attribute>
     <widget class="QPushButton" name="Runner" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Run</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3" >
      <property name="geometry" >
       <rect>
        <x>680</x>
        <y>20</y>
        <width>21</width>
        <height>181</height>
       </rect>
      </property>
      <property name="cursor" >
       <cursor>13</cursor>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text" >
       <string>X</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="MacroSave" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>400</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save As Macro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changer" >
      <property name="geometry" >
       <rect>
        <x>310</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Change Dir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4" >
      <property name="geometry" >
       <rect>
        <x>600</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5" >
      <property name="geometry" >
       <rect>
        <x>680</x>
        <y>220</y>
        <width>16</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text" >
       <string>X</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2" >
      <property name="geometry" >
       <rect>
        <x>110</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Clear All</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="KonsoleEm" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>220</y>
        <width>651</width>
        <height>171</height>
       </rect>
      </property>
      <property name="cursor" >
       <cursor>4</cursor>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Display" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>10</y>
        <width>651</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="Bar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>saveras</tabstop>
  <tabstop>ChangeFile</tabstop>
  <tabstop>CommandLine</tabstop>
  <tabstop>saver</tabstop>
  <tabstop>MotorsTree</tabstop>
  <tabstop>Tabby</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>ClearLog</tabstop>
  <tabstop>MaxVal</tabstop>
  <tabstop>Responses</tabstop>
  <tabstop>SaveImage</tabstop>
  <tabstop>MinVal</tabstop>
  <tabstop>Zoomer</tabstop>
  <tabstop>Element</tabstop>
  <tabstop>EStop</tabstop>
  <tabstop>KonsoleEm</tabstop>
  <tabstop>Runner</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MotorHead</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>74</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel" >
     <x>165</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MaxVal</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MaxValeDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>692</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel" >
     <x>712</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MinVal</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MinValeDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>632</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel" >
     <x>642</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Zoomer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>672</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel" >
     <x>722</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>KonsoleEm</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>128</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MotorHead</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>618</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel" >
     <x>507</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Display</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>692</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel" >
     <x>655</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Display</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>90</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>KonsoleEm</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>700</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel" >
     <x>664</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Positioner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MoveBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>517</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel" >
     <x>541</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MoveBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Positioner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>537</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>496</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VidZoom</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>273</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel" >
     <x>355</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VidZoom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>396</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
